src/sparseml/transformers/question_answering.py

```diff
@@ -67,6 +67,18 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+metadata_args = [
+    "distill_hardness",
+    "distill_temperature",
```

Comment:
these args actually should no longer be used (just took a quick look at script, please double check), if you could remove the args in the script, would be great. (these args are now controlled in the recipe!)

Reply:
This is correct.